tools/webpack/shared.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+const TerserPlugin = require( 'terser-webpack-plugin' );
+const postcss = require( 'postcss' );
+const { join } = require( 'path' );
+
+const baseDir = join( __dirname, '../../' );
+
+const baseConfig = ( env ) => {
+	const mode = env.environment;
+
+	const config = {
+		target: 'browserslist',
+		mode,
+		optimization: {
+			moduleIds: mode === 'production' ? 'deterministic' : 'named',
+			minimizer: [
+				new TerserPlugin( {
+					extractComments: false,
+				} ),
+			]
+		},
+		module: {
+			rules: [
+				{
+					test: /\.js$/,
+					use: [ 'source-map-loader' ],
+					enforce: 'pre',
+				},
+			],
+		},
+		resolve: {
+			modules: [
+				baseDir,
+				'node_modules',
+			],
+			alias: {
+				'lodash-es': 'lodash',
+			},
+		},
+		stats: 'errors-only',
```

Comment:
Not sure why v5 shows alot more stats, but changed it to only show errors. From the logs I only see v4 logging an hash and the webpack version.

Reply:
Should we include also warnings: `'errors-warnings'`? Unless it's too noisy. We use the default value in Gutenberg.